### ISSUE TITLE
Echo Arena control issue

### ISSUE THREAD

**Issue by Mr-Ced** [LibreVR/Revive#646]
Echo Arena/Lone Echo relies a lot on the grip buttons, atm Revive puts the grip on a toggle on/off when pressed quickly and that's a huge problem for this game.

Because of this issue, we can't properly interact in the game like throwing the disc as soon as we grab it, push ourselves from a wall/person quickly.

Final point, the toggle makes it too hard to know for this type of game whether or not the interaction with the environment/players is on and gets messy really fast.

Tldr : The grip toggle on/off needs to be deactivated for this game

**Comment by Armada651** [comment]
You can adjust the grip settings for Revive in [OpenVR-AdvancedSettings](https://github.com/matzman666/OpenVR-AdvancedSettings).

I'm considering going back to the old default setting that never toggles.

**Comment by Mr-Ced** [comment]
Well, never mind then. ^^
Sorry for the waste of time.
